recipes/dbsp_drp/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "DBSP_DRP" %}
+{% set version = "0.9.0" %}
+{% set pypi_name = "dbsp-drp" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7308095ae3709445be11b4e488d4c113b02ddd298daa8607be76e1f71e547b6f
+
+build:
+  skip: True  # [win or py<38]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm >=5
+  run:
+    - python
+    - numpy >=1.16.0
+    - astropy >=4.0
+    - scipy >=1.4
+    - matplotlib >=3.1
```

Comment:
Since you're installing pyside2 explicitly, better to depend on `matplotlib-base`, since for conda-forge, `matplotlib` is just `matplotlib-base` + `pyqt`.

Reply:
thanks for catching that!